src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -34,11 +34,15 @@
    :on-press            #(rf/dispatch [:open-modal :screen/wallet.share-address {:status :receive}])})
 
 (defn- action-bridge
-  []
+  [token-data]
   {:icon                :i/bridge
    :accessibility-label :bridge
    :label               (i18n/label :t/bridge)
-   :on-press            #(js/alert "to be implemented")})
+   :on-press            (fn []
+                          (rf/dispatch [:hide-bottom-sheet])
+                          (rf/dispatch [:wallet/bridge-select-token
```

Comment:
You should be using the bridging wizard instead and should add a new flow for bridge

Reply:
I agree @mmilad75, this is a ideal use case for wizard navigation.

Reply:
@mmilad75 fixed it. Could you please have a look at the code changes. 

And here is a video:

https://github.com/status-im/status-mobile/assets/29354102/82e9dbcf-a495-44f4-a59d-41f7ac1ff3c0

Reply:
Nice. Thanks man 🙏